src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,8 @@
+@mixin generalProperties {
```

Comment:
Will be better to use more clear name for the mixin
I can't get from the name for what element and for what purpose I can add it